src/_qhull_wrapper.cpp

```diff
@@ -204,12 +214,12 @@ delaunay_impl(npy_intp npoints, const double* x, const double* y,
     std::vector<int> tri_indices(max_facet_id+1);
 
     /* Allocate Python arrays to return. */
-    npy_intp dims[2] = {ntri, 3};
-    numpy::array_view<int, ndim> triangles(dims);
-    int* triangles_ptr = triangles.data();
+    int dims[2] = {ntri, 3};
+    IndexArray triangles(dims);
```

Comment:
You don't even need to allocate dims on the stack; I usually just write `IndexArray triangle{{ntri, 3}};`.  Ditto below.

Reply:
I know. Remember that the idea of this PR is to demonstrate the new approach works and show how to map from the old to the new implementation. There used to be an explicit line declaring the `dims`, so the new code has the equivalent line declaring the `dims`.